tests/test_logic.py

```diff
@@ -273,7 +273,7 @@ def test_Client_on_authenticate_failed(homedir, config, mocker):
     mock_gui = mocker.MagicMock()
     mock_session = mocker.MagicMock()
     cl = Client('http://localhost', mock_gui, mock_session, homedir)
-    result_data = 'false'
+    result_data = Exception('oh no')
```

Comment:
and this is a better test since the failure case is there being an `AuthError` :+1: